### DISCUSSION TITLE
Disable CartesianMarker in specific situations

### DISCUSSION THREAD

**Discussion by Itzekey** [patrykandpatrick/vico#973] [Questions]
### Question

Hello, I would like to know if it's possible to disable the marker (I'm using "public interface CartesianMarker : ChartInsetter<CartesianChartModel>"), when e.g. the y-value at a certain x-value is 0.

### Vico version(s)

2.0.0-alpha.25

### UI framework(s)

Jetpack Compose

**Comment by Gowsky** [comment]
Hello, @Itzekey. Simply do an early return from the drawing function in your custom `CartesianMarker` implementation.
